### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by Akimaru666** [kuglee/TermiWatch#30]
When I go to project > run, it let me an error:

WatchKit App doesn't contain any WatchKit Extensions whose WKAppBundleIdentifier matches "com.(me).TermiWatch". Verify that the value of WKAppBundleIdentifier in your WatchKit Extension's Info.plist matches the value of CFBundleIdentifier in your WatchKit App's Info.plist.

what can I doo ?

**Comment by kuglee** [comment]
Please see issue #9 .

**Comment by Akimaru666** [comment]
Yeah but when I do cmd+f and I search kuglee, I see nothing to replace with my apple indent

**Comment by kuglee** [comment]
These are the places where you have to change to identifiers:

- TermiWatch WatchKit App's Info.plist:
WKCompanionAppBundleIdentifier

- TermiWatch WatchKit Extension's Info.plist:
NSExtension > NSExtensionAttributes

**Comment by Akimaru666** [comment]
Thanks for helping me, I look at both theses places but I've already replace them :( still don't work

**Comment by kuglee** [comment]
Try to clean your project (Xcode > Product menu > Clean Build Folder).

**Comment by Akimaru666** [comment]
done, still don't work

**Comment by kuglee** [comment]
Do you have ```com.(me).TermiWatch.watchkitapp``` in NSExtension > NSExtensionAttributes > WKAppBundleIdentifier?

Also, select the project in the Navigator and check it the Target have the correct Bundle Identifiers.

**Comment by Akimaru666** [comment]
I have this: com.(me).TermiWatch.watchkitapp, but I don't know how to "check it"

**Comment by kuglee** [comment]
It's in the How to Install guide > Configuring the project section.
The targets have to have the following bundle identifiers.

- TermiWatch: com.(me).TermiWatch
- TermiWatch WatchKit App: com.(me).TermiWatch.watchkitapp
- TermiWatch WatchKit Extension: com.(me).TermiWatch.watchkitapp.watchkitextension

**Comment by Akimaru666** [comment]
IT WORKED ! I was missing this one com.(me).TermiWatch.watchkitapp, thank you so much for helping me so far <3

**Comment by kuglee** [comment]
No problem. Glad I could help.

**Comment by wangqiaowen** [comment]
@kuglee I'm still getting the same error after checking 3 bundle identifiers...
